### PR TITLE
Ingestion: ICNLSP 2023

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by anthology-assist** [comment]
closes #2879

File: data/xml/2023.icnlsp.xml [hunk 1]
```diff
@@ -0,0 +1,350 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.icnlsp">
+  <volume id="1" ingest-date="2023-12-31" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of The sixth International Conference on Natural Language and Speech Processing (ICNLSP-2023)</booktitle>
```
Comment (mjpost) [inline data/xml/2023.icnlsp.xml:5]: Weird capitalization, also title should be made consistent with previous volumes: 
image

Please see [ingestion checklist](https://github.com/acl-org/acl-anthology/wiki/Ingestion-Checklist)

Comment (icnlsp) [reply data/xml/2023.icnlsp.xml:5]: Proceedings of the 6th International Conference on Natural Language and Speech Processing (ICNLSP-2023)

**Review by icnlsp — CHANGES_REQUESTED** [review]
Proceedings of the 6th International Conference on Natural Language and Speech Processing (ICNLSP-2023)

**Review by icnlsp — APPROVED** [review]

File: data/xml/2023.icnlsp.xml [hunk 2]
```diff
@@ -0,0 +1,350 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.icnlsp">
+  <volume id="1" ingest-date="2023-12-31" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of The 6th International Conference on Natural Language and Speech Processing (ICNLSP 2023)</booktitle>
```
Comment (mjpost) [inline data/xml/2023.icnlsp.xml:5]: The → the

**Pull request merged**